app/(login)/verify-email.tsx

```diff
@@ -0,0 +1,17 @@
+import { PrivyProvider } from '@privy-io/expo';
+import { VerifyEmail } from '@/components';
+import { useLocalSearchParams } from 'expo-router';
+
+const LoginVerifyEmailScreen = () => {
+  const params = useLocalSearchParams();
+
+  const email = params?.email;
+
+  return (
+    <PrivyProvider appId={process.env.EXPO_PUBLIC_PRIVY_APP_ID!} clientId={process.env.EXPO_PUBLIC_PRIVY_CLIENT_ID}>
```

Comment:
Don't repeat yourself.

I see 6 privy providers spread across different files.

Reply:
Also, make sure

It's not a good practice to use the `!` TS operator.

Reply:
@renanmav just deleted some of those files. Currently using the Provider in 4 files. Question; how do you think we should handle this? can't put it in the `_layout.tsx` file cos it throws an error.

Reply:
Also, the envs are set properly. I included a `!` because privy thinks `process.env.EXPO_PUBLIC_PRIVY_APP_ID` could be undefined.

Reply:
done these. I guess layout was failing prev cos i needed to rebuild